WordPressLint/src/test/java/org/wordpress/android/lint/MissingNullAnnotationDetectorTest.kt

```diff
@@ -0,0 +1,109 @@
+package org.wordpress.android.lint
+
+import com.android.tools.lint.checks.infrastructure.LintDetectorTest
+import com.android.tools.lint.checks.infrastructure.TestLintTask.lint
+import org.junit.Test
+
+
+class MissingNullAnnotationDetectorTest {
+    @Test
+    fun `it should inform when null annotation is missing on field`() {
+        lint().files(LintDetectorTest.java("""
+            package test;
+
+            class ExampleClass {
+              String mExampleField = "example";
+            }
+        """).indented())
+                .allowCompilationErrors()
```

Comment:
Minor (🔍): Consider removing this `allowCompilationErrors()` since all tests are working even without allowing for compilation errors.

Reply:
Resolved in: https://github.com/wordpress-mobile/WordPress-Lint-Android/pull/11/commits/7295d2a315670cd0b1c27ed93ba9e6002f3da252

Reply:
Awesome, thanks for this update @mkevins ! 💯